docs/page.rst

```diff
@@ -974,22 +976,27 @@ In a nutshell, this is what you can do with PyMuPDF:
       This is a **global replacement:** the new image will also be shown wherever the old one has been displayed throughout the file.
 
       This method mainly exists for technical purposes. Typical uses include replacing large images by smaller versions, like a lower resolution, graylevel instead of colored, etc., or changing transparency.
+
+      The page is not "aware" of this replacement and will treat the new image like it did the old one, so for example the same bbox will be used.
    
    
    .. index::
       pair: xref; delete_image
 
    .. method:: delete_image(xref)
 
-      * New in v1.21.0
+      * New in v1.21.1
 
-      Delete the image at xref. This is slightly misleading: actually the image is being replaced with a small transparent :ref:`Pixmap` using above :meth:`Page.replace_image`. The visible effect however is equivalent.
+      Delete the image at xref. This is slightly misleading: actually the image is being replaced with a small transparent :ref:`Pixmap` using above :meth:`Page.replace_image`. The visible effect is as if there were no image.
 
       :arg int xref: the :data:`xref` of the image.
 
-      This is a **global replacement:** the image will disappear wherever the old one has been displayed throughout the file.
+      This is a **global "deletion":** the image will disappear wherever the old one has been displayed throughout the file.
+
+      If you inspect / extract a page's images by methods like :meth:`Page.get_images`, :meth:`Page.get_image_info` or :meth:`Page.get_text`, the replacing "dummy" image will be detected like so `(45, 47, 1, 1, 8, 'DeviceGray', '', 'Im1', 'FlateDecode')` and also seem to "cover" the same boundary box on the page.
    
    
+>>>>>>> Stashed changes
```

Comment:
This looks wrong to me?

Reply:
yes, it is wrong, I therefore changed the state to draft.

Reply:
aha - okay - I checked over all the RST stuff, and that was more or less all I found - but I'll look again once this comes out of "Draft".